.github/workflows/build.yml

```diff
@@ -9,14 +9,21 @@ jobs:
   build:
     runs-on: ubuntu-18.04
     steps:
-    - name: checkout
-      uses: actions/checkout@v2
-
+    
     - uses: actions/setup-go@v2
       with:
         go-version: 1.16
 
+    - name: checkout
```

Comment:
Why changing the position of `checkout`?

Reply:
this what followed in other action. example codegen, crds-gen